config/galaxy.ini

```diff
@@ -846,12 +1025,19 @@ use_interactive = True
 # to True to force these to lower case.
 #normalize_remote_user_email = False
 
+# If an e-mail address is specified here, it will hijack remote user mechanics
+# (``use_remote_user``) and have the webapp inject a single fixed user. This
+# has the effect of turning Galaxy into a single user application with no
+# login or external proxy required. Such applications should not be exposed to
+# the world.
+#single_user =
+
 # Administrative users - set this to a comma-separated list of valid Galaxy
 # users (email addresses).  These users will have access to the Admin section
 # of the server, and will have access to create users, groups, roles,
 # libraries, and more.  For more information, see:
-#   https://wiki.galaxyproject.org/Admin/Interface
-admin_users = phenomenal@phenomenal-h2020.eu
+#   https://galaxyproject.org/admin/
```

Comment:
This seems safe according to https://github.com/phnmnl/container-galaxy-k8s-runtime/blob/c315689a9def5628c482fae170b4c68240527887/ansible/configure_galaxy.py#L151